Refresh.GameServer/Database/GameDatabaseProvider.cs

```diff
@@ -70,6 +70,7 @@ protected GameDatabaseProvider(IDateTimeProvider time)
         typeof(GameReview),
         typeof(DisallowedUser),
         typeof(RateReviewRelation),
+        typeof(TagLevelRelation),
```

Comment:
no db schema bump?

Reply:
realm doesn't need it for just adding new types, it only matters when an existing type changes or gets removed(?)